assets/controllers/login-controller.js

```diff
@@ -0,0 +1,44 @@
+// A Stimulus JavaScript controller file
+// https://stimulus.hotwired.dev
+// @see templates/security/login.html.twig
+// Leverage the Symfony Stimulus Bridge
+// https://github.com/symfony/stimulus-bridge
+// More info on Symfony UX
+//https://github.com/symfony/ux
```

Comment:
I think we could merge these last 4 lines (starting with "Leverage") into just this last line (the link to UX). But with a space between `//` and the `https`.